transfer.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+package rusk;
+option go_package = "github.com/dusk-network/rusk-schema;rusk";
+
+import "transaction.proto";
+import "keys.proto";
+
+message TransferTransactionRequest {
+    fixed64 value = 1;
+    StealthAddress recipient = 2;
```

Comment:
Should Golang specify also the `type` and `version`? @ZER0

Reply:
No, as discussed previously, the node cannot "request" a specific version since we assumed that the input and the returned value would be the same across all the versions, and it's not a valid assumption.
Also, the "type" is always the same for those transactions, we cannot have a different type for a transfer tx.